commit 37fa0b13a2503039ea0e9ba785f1951785d941ad (refs/changes/61/1761/3)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2017-04-10T21:06:17+00:00 (2 years, 6 months ago)
    
    OS-5565 Need smrt HBA mode support
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/overlay/generic/etc/driver_classes b/overlay/generic/etc/driver_classes
index 8608747a..3124f9a2 100644
--- a/overlay/generic/etc/driver_classes
+++ b/overlay/generic/etc/driver_classes
@@ -27,4 +27,4 @@ qlc	fibre-channel
 si3124	scsi-self-identifying
 dr_sas	scsi-self-identifying
 skd	disk
-smrt	scsi
+smrt	scsi-self-identifying
